ovos_tts_plugin_server/__init__.py

```diff
@@ -28,8 +28,18 @@ def __init__(self, *args, **kwargs):
 
     @property
     def host(self) -> Optional[str]:
+        self.log.warning("Deprecation Warning: use self.urls instead")
+        return self.urls
```

Comment:
looks like this is returning a list but expects a string, maybe `self.urls[0]`?

Reply:
I'm also not super clear on the intention here - this is a breaking change because `self.host` is no longer referenced anywhere in the code. You're now setting self.urls as the return value, but self.urls is actually pulling from `hosts`.

Actually now I see - host is still referenced but nested under self.host.

This is pretty confusing. If we're going to create a new configuration parameter, we should make the property and the parameter name match if we can for clarity. It's going to be very hard to test and challenging for devs to read the way it is now.